commands/alias.go

```diff
@@ -36,7 +36,7 @@ func alias(command *Command, args *Args) {
 	}
 
 	if shell == "" {
-		utils.Check(fmt.Errorf("Unknown shell"))
+		utils.Check(fmt.Errorf("Unknown shell: Set the SHELL enviornment variable or execute 'hub alias -s <shell name>'"))
```

Comment:
No, you run `hub alias` shell to get the instructions, and `hub alias -s` to get the code output suitable for eval'ing. Basically, `-s` opts into eval mode and isn't related to `<shell>` argument at all.